src/chart/table/TableChart.tsx

```diff
@@ -28,8 +27,8 @@ import Button from '@mui/material/Button';
 import { renderCellExpand } from '../../component/misc/DataGridExpandedRenderer';
 
 const TABLE_HEADER_HEIGHT = 32;
-const TABLE_FOOTER_HEIGHT = 52;
-const TABLE_ROW_HEIGHT = 78;
+const TABLE_FOOTER_HEIGHT = 62;
```

Comment:
Brahm commented the TABLE_FOOTER_HEIGHT variable as it is no longer used. Previously we were using it for calculating dynamic rowHeight. Which we are not going to do hereafter since we are using density.